tests/yam/agama/import_agama_profile.pm

```diff
@@ -7,14 +7,13 @@
 use base Yam::Agama::patch_agama_base;
 use strict;
 use warnings;
-use testapi qw(assert_script_run data_url get_required_var select_console script_run);
+use testapi qw(diag assert_script_run data_url get_required_var select_console script_run);
 
 sub run {
     select_console 'root-console';
     my $profile = get_required_var('AGAMA_PROFILE');
-    my $profile_url = data_url($profile);
     script_run("dmesg --console-off");
-    assert_script_run("/usr/bin/agama profile import $profile_url", timeout => 300);
+    assert_script_run("/usr/bin/agama profile import $profile", timeout => 300);
```

Comment:
could you check if `/usr/bin/` is needed here? I wanted to removed the other day but didn't have the time.

Reply:
Yes, /usr/bin not needed. https://openqa.suse.de/tests/16404499#step/import_agama_profile/10